### PR TITLE
Disable multitenancy for non-production environments

### PR DESCRIPTION
**Pull request by cbeer**
This may make it easier to develop and test against the application.

### CODE REVIEW

File: spec/requests/home_spec.rb [hunk 1]
```diff
@@ -6,6 +6,10 @@
   end
 
   context 'without a current tenant' do
+    before do
+      allow(ApplicatioController).to receive(:multitenant?).and_return(true)
```
Comment (mjgiarlo) [inline spec/requests/home_spec.rb]: `s/ApplicatioController/ApplicationController/`

Also, you're using `#multitenant?` here and `#multitenant` (sans `?`) above on line 19 of the `ApplicationController`.

Comment (cbeer) [inline spec/requests/home_spec.rb]: Thanks. Pretty weird that nothing caught that.

Comment (mjgiarlo) [inline spec/requests/home_spec.rb]: Travis did!

**Comment by mjgiarlo** [comment]
:+1:, if Travis is :green_heart:

**Pull request merged**